src/Pages/Sign/Terms.scss

```
@@ -0,0 +1,50 @@
+.termsBox {
+  border-top: 2px solid #a36d6a;
+  padding-bottom: 60px;
+  .termTitleBox {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 60px;
+    margin-bottom: 30px;
+    .termTitle {
+      color: #8f5b59;
+      font-size: 1.17em;
+    }
+    .checkBox {
+      display: flex;
+      align-items: center;
+      .check {
+        width: 15px;
+        height: 15px;
+        margin-right: 8px;
+        background-color: #a36d6a;
+      }
+      .agree {
+        color: #8f5b59;
+        font-size: 13px;
+        font-weight: 700;
+        padding-right: 10px;
+      }
+    }
+  }
+  .agreementContent {
+    color: #333333;
```

6자리가 모두 동일할 경우에는 3자리까지만 써줘도 결과는 같습니다
```suggestion
    color: #333;
```